crane-cloud/frontend
# Added correct data for the apps and databases count for a user

# Description

- Added correct counts for the apps and databases for a specific user
- Removed controls for admin account

## Type of change


## Trello Ticket ID

https://trello.com/c/q6kU3ySa

## How Can This Been Tested?

Run this branch locally then access the admin account under the users listing to see the changes.

## Checklist:


# Screenshots

![Screenshot from 2023-09-26 16-21-17](https://github.com/crane-cloud/frontend/assets/108899937/019ac77a-f546-46da-b6dd-03297ffee479)

![Screenshot from 2023-09-26 16-22-44](https://github.com/crane-cloud/frontend/assets/108899937/9dc8ffc8-374f-48b6-9fbe-1c268ea75958)
